plugin.video.s04tv/resources/language/English/strings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
xml format is seriously deprecated. Can you move to strings.po?
Also the folder layout should be `/resources/language/resource.language.en_gb/strings.po`